darwin/datatypes.py

```diff
@@ -32,17 +32,50 @@ def get_sub(self, annotation_type: str) -> Optional[SubAnnotation]:
                 return sub
 
 
+@dataclass(frozen=True, eq=True)
+class VideoAnnotation:
+    annotation_class: AnnotationClass
+    frames: Annotation
+    keyframes: List[bool]
+    segments: List[List[int]]
+    interpolated: bool
+
+    def get_frame(self, frame_index: int):
+        return frames[frame_index]
+
+    def get_data(self, only_keyframes=True, post_processing=None):
```

Comment:
It would be great if we could add type annotations here as well. I think the following makes sense:
```suggestion
    def get_data(
      self,
      only_keyframes: bool = True,
      post_processing: Optional[Callable[[Annotation, dict], dict]] = None,
    ):
```